### PR TITLE
add redrun

### PR DESCRIPTION
**Pull request by coderaiser**
Redrun
Added [redrun](https://github.com/coderaiser/redrun): ✨🐌🐎✨ Expand `script` from `package.json` to improve execution speed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Tools for running multiple commands or npm scripts in parallel or sequentially.
 
 - [script-runner](https://github.com/paulpflug/script-runner) - Simple task runner with a terse syntax.
 - [npm-run-all](https://github.com/mysticatea/npm-run-all) - Fully featured task runner.
+- [redrun](https://github.com/coderaiser/redrun) - expand scripts from package.json to improve execution speed.
```
Comment (RyanZim) [inline README.md]: `expand` => `Expand`; should be capitalized.

**Review by RyanZim — CHANGES_REQUESTED** [review]
Minor capitalization nitpick, other than that, LGTM.

**Comment by RyanZim** [comment]
Also at https://github.com/coderaiser/redrun/blob/master/README.md#completion:

> # Completion
> 
> You can enables tab-completion of npm scripts 

Should be "You can enable tab-completion". Just something I noticed.

**Comment by coderaiser** [comment]
Thank you, fixed in 014b11f.

**Review by RyanZim — APPROVED** [review]

**Comment by RyanZim** [comment]
Thanks!

**Pull request merged**